tabbycat/utils/tables.py

```diff
@@ -370,19 +385,7 @@ def _result_cell_two(self, ts, compress=False, show_score=False, show_ballots=Fa
             cell['popover']['content'].append(
                 {'text': _("Total speaker score: <strong>%s</strong>") % metricformat(score)})
 
-        if show_ballots:
-            if self.admin:
-                cell['popover']['content'].append({
-                    'text': _("View/edit debate ballot"),
-                    'link': reverse_tournament('old-results-ballotset-edit',
-                            self.tournament, kwargs={'pk': ts.ballot_submission_id}),
-                })
-            elif self.tournament.pref('ballots_released'):
-                cell['popover']['content'].append({
-                    'text': _("View debate ballot"),
-                    'link': reverse_tournament('results-public-scoresheet-view',
-                            self.tournament, kwargs={'pk': ts.debate_team.debate_id}),
-                })
+        self._show_ballots(show_ballots, cell, reverse_tournament, ts)
```

Comment:
Calling this function implies that we wish to show ballots. The `show_ballots` conditional should be here and the other call, not in the new method.